src/layout/MainLayout/MainLayout.styles.tsx

```diff
@@ -0,0 +1,7 @@
+import { styled } from '@/stitches.config'
+
+export const MainWrapper = styled('div', {
+  display: 'grid',
+  gridTemplateRows: 'auto 1fr auto',
+  minHeight: '100vh'
+})
```

Comment:
I would like to have a chat to discuss this way of having the MainLayout. This way you will always need 3 elements on your layout, but imagine if you don't need the navbar or the footer. I think that with flex is more versatile

Reply:
We can have a call if you want 

<img src="https://media2.giphy.com/media/3yRHVLXGeodopGpq7l/giphy.gif"/>